portal-frontend/src/app/features/applications/edit-submission/primary-contact/primary-contact.component.html

```diff
@@ -1,92 +1,116 @@
-<div class="step-description">
-  <h2>Primary Contact</h2>
-  <p>
-    The primary contact becomes the single point of contact and must ensure that landowners are notified of all
-    information exchanged between the primary contact, local or First Nation government, and the ALC.
-  </p>
-  <p>*All fields are required unless stated optional or disabled.</p>
-</div>
+<section>
+  <div class="step-description">
+    <h2>Primary Contact</h2>
+    <div>
+      <p>
+        The primary contact becomes the single point of contact and must ensure that landowners are notified of all
+        information exchanged between the primary contact, local or First Nation government, and the ALC.
+      </p>
+    </div>
+    <div><p>*All fields are required unless stated optional or disabled.</p></div>
```

Comment:
Nitpick: What is the reason for all the wrapping divs? Same for NOI

Reply:
You are correct. They are useless. I added these, because originally these paragraphs were pushed up against each other. I was trying to understand the layout system, so I looked around at other pages. One of them had the paragraphs wrapped in divs and the paragraphs had margins, so I thought it might be a scoping issue, but it turned out these were added in a custom CSS rule for that page. My issue was that the entire step-description contents were wrapped in a div, which was preventing the flex gap. I just forgot to take those divs out.